core-bundle/src/Controller/BackendController.php

```diff
@@ -156,6 +156,17 @@ public function pickerAction(Request $request): RedirectResponse
         return new RedirectResponse($picker->getCurrentUrl());
     }
 
+    #[Route('/{parameters}', name: 'contao_backend_fallback', requirements: ['parameters' => '.*'], defaults: ['statusCode' => 404], priority: -1000)]
```

Comment:
The `'statusCode' => 404` does not work here I think. 
It has to be handled inside the method.

Reply:
You're right, I noticed this earlier but didn't get a chance to fix it. 😬 Should I open another PR?

Reply:
Yes, please. 🙏

Reply:
`return $this->render(…, …, new Response('', 404));` should do the trick I think.